.golangci.yml

```diff
@@ -0,0 +1,14 @@
+linters-settings:
+  wsl:
+    allow-cuddle-declarations: true
+
+issues:
+  exclude-rules:
+    - linters: ["gocritic"]
+      text: "ifElseChain:"
+    - linters: ["gosec"]
+      text: "G204:"
+      path: util.go
```

Comment:
If it is just for util.go its better to use //nolint:gosec on the offending line, that way while reading code its apparent that we are suppressing that and we know the context/reason.

Reply:
yeah, i guess you're right here.  I really do nnont like '//nolint' lines generally, but this really is one line t hat we're saying is OK for this.